apps/meteor/app/api/server/v1/videoConference.ts

```diff
@@ -23,3 +26,27 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'video-conference.start',
```

Comment:
I dont think we need `.start`, since we are using `POST` [POST] video-conference should be enough

Reply:
This is the standard for the v1 API.
v2 will be RESTful